src/sentry/static/sentry/app/stores/groupingStore.tsx

```diff
@@ -45,12 +45,6 @@ type State = {
 
 type ScoreMap = Record<string, number | null>;
 
-type Item = {
```

Comment:
Since `Item` is also used by `mergedList`.

Reply:
I would probably export this from here (and maybe rename it as `Item` is pretty generic) since this type will probably only be used by consumers of this store, and not generically used outside of it.